dataprofiler/profilers/profile_builder.py

```diff
@@ -1201,8 +1200,12 @@ def __add__(self, other):
         # merge profiles
         for profile_name in self._profile:
             merged_profile._profile[profile_name] = (
-                self._profile[profile_name] + other._profile[profile_name]
+                    self._profile[profile_name] + other._profile[profile_name]
             )
+
+        # merge correlation
+        merged_profile.correlation_matrix = self._merge_correlation(other)
```

Comment:
is this feature not being turned off till we solve the issues we discussed?

Reply:
as above